tinygrad/tensor.py

```diff
@@ -1033,16 +1033,17 @@ def shrink(self, arg:tuple[Optional[tuple[sint, sint]], ...]) -> Tensor:
   def pad(self, padding:Union[Sequence[sint], Sequence[Optional[tuple[sint, sint]]]], mode:str="constant", value:float=0.0) -> Tensor:
     """
     Returns a tensor with padding applied based on the input `padding`.
+
     `padding` supports two padding structures:
 
-    1. Flat padding: (padding_left, padding_right, padding_top, padding_bottom, ...)
-       - This structure matches PyTorch's pad.
-       - `padding` length must be even.
+    1. Flat padding: `(padding_left, padding_right, padding_top, padding_bottom, ...)`
+        - This structure matches PyTorch's pad.
+        - `padding` length must be even.
 
-    2. Group padding: (..., (padding_top, padding_bottom), (padding_left, padding_right))
-       - This structure matches pad for jax, numpy, tensorflow and others.
-       - For each axis, padding can be `None`, meaning no padding, or a tuple `(start, end)`.
-       - `padding` must have the same length as `self.ndim`.
+    2. Group padding: `(..., (padding_top, padding_bottom), (padding_left, padding_right))`
+        - This structure matches pad for JAX, NumPy, TensorFlow, and others.
+        - For each axis, padding can be `None`, meaning no padding, or a tuple `(start, end)`.
+        - `padding` must have the same length as `self.ndim`.
```

Comment:
This wasn't rendering properly for mkdocs (was enumerating to 7...)
need 4 spaces for sub bullet points
need to remind myself this is markdown

also btw there are some idiosyncrasies to markdown I don't think the other documentation are aware of
e.g. the + at line 1036 breaks the line into two. Two adjacent lines without two whitespaces at end of first line will not line break

<img width="1101" alt="image" src="https://github.com/user-attachments/assets/d4e5be76-738c-4206-856f-e4820a580c0c" />
was this in tensor.py

![image](https://github.com/user-attachments/assets/92f3227b-f2f8-4e93-b919-9590dbf4157b)